### ISSUE TITLE
Venue for 2017-2018

### ISSUE THREAD

**Issue by gsaslis** [devstaff-crete/DevStaff-Heraklion#185]
I'm opening this issue for a general discussion on whether we should move away from our venue in ITE/FORTH, so that all concerns and pros/cons can be voiced and recorded, for future reference. 

So far, most of the discussions have taken place in a rather non-persistent form, so I think there's value in keeping a central reference as this issue has come up more than once. This would serve as a good place to send ppl next time it does.

For reference, I am recording some key requirements for our venue of choice, as collected over the past 2 years: 

- [ ] must be free: always having to find sponsors to cover the venue is a huge issue, we want to avoid it, given that we have alternatives. 
- [ ] must be big enough for 50-100 ppl: this is the range our attendance 
- [ ] projector + screen must be available
- [ ] decent internet connection (for remote speakers)

So far, we are aware of two such venues: 

:one: ITE/FORTH 
:two: Koinonikos Horos, Idrymata Kalokairinou @ town center

I am also recording some pros/cons for each one, that we have discussed in past org meetups.

### Pros
- ample parking space
- great internet speeds
- almost always available
- it has become the de facto DevStaff meetup venue

### Cons
- You probably need a car to get there. 
- Even with a car, it does take some time (though maybe not more than trying to find a parking spot in town)
- limited capacity: when over 80ppl it gets really crammed

### Pros
- walking distance for many people 
- by donating food supplies DevStaff gives back something to the broader community, not just fellow devs
- bigger capacity (100-120ppl)
- post-event activities: in town, so lots of options (food/drinks)

### Cons
- Sound: should be fine with microphone, but suffers otherwise
- projector: poor quality
- networking space: not as big
- parking is a problem
- internet: wifi was poor, not sure about speed of ethernet connectivity

_if more venue options are added in the comments below, I can update this list, so we can use corresponding reactions as votes. Still, this is not just a vote - it is an open board for discussion and sharing your views_

**Comment by maounis** [comment]
(First of all, thank you George for taking the time to write down those issues)

I vote for ITE/FORTH. 

This does not mean that we can't keep donating food supplies to the broader community if we can.
I am sure someone is living nearby to deliver them, otherwise I will.

**Comment by nickorfas** [comment]
I vote for ITE/FORTH mainly because of parking issues in the center of the town on Thursday evenings.

**Comment by padeler** [comment]
Good idea putting the venue in the form of an issue George, this way it is a more permanent discussion. 

I also prefer FORTH because of the parking and its proximity to my office. 

On the other hand there are people who want to attend but cannot come to FORTH. So if anyone has a suggestion for a place that is a good compromise between the two available venues, it would be great to try it out.

**Comment by markop** [comment]
One more vote for ITE/FORTH, from me :)

Besides the parking space, which is important, and the proximity to my home as well as my office, there is one more reason I prefer it: it just "feels" more appropriate as a venue in my mind...

**Comment by irinikp** [comment]
I vote for ITE/FORTH and I agree that we can keep donating either way. 

Also regarding the time needing to get there, for me it seems less than the time I need to reach Koinonikos Horos. Also the projector quality was poor.

As for the people who want to attend but can't come to FORTH, I volunteer to transfer 2-3 if we agree on a meeting point. If more people can do that and we properly organize it, it can work

**Comment by sonaht** [comment]
On the principle of offering to the community while doing our thing I would have loved to say that Koinonikos Horos is a good option but unfortunately it lacks some of the so much needed amenities.

So I vote for ITE/FORTH and I agree that we must keep donating one way or another.

**Comment by svelon** [comment]
I prefer ITE/FORTH than Koinonikos Horos, Idrymata Kalokairinou. I would like to have more options though. I cannot suggest anything else now, but I'm sure there are other places, too.

And yes, we should keep donating. Perhaps colaborate with specialized groups for that (if not with Koinonikos Horos directly)?

**Comment by sksksksk** [comment]
Coming each time from Chania, I prefer ITE because of the easy parking and the connection to the national road

**Comment by kabitakis** [comment]
One more vote for ITE/FORTH

**Comment by webdevgr** [comment]
+1 ITE/FORTH

**Comment by zakkak** [comment]
Whatever makes org team's life easier. I guess that's FORTH

**Comment by jamiedarcher** [comment]
COMEET has a great basement space in town would probably be big enough to host a meeing. Not sure about parking but they do film some screenings there on Wednesdays so the projector is probably pretty good.
https://web.facebook.com/ComeetCreativeSpace/?_rdc=1&_rdr

**Comment by svelon** [comment]
@jamiedarcher Comeet is awesome, however it does have a parking issue and the space can host up to 90 people max, while some times we are more than 100.

**Comment by gsaslis** [comment]
@jamiedarcher @svelon we could give it a try for one of the upcoming meetups. 

Do you know if there would be a cost for us going there? Does someone have an existing contact there to explore this?

**Comment by estavrak** [comment]
another vote for ITE

**Comment by jamiedarcher** [comment]
@gsaslis I'm enquiring about prices now for Comeet. I have looked at parking the only thing that looks suitable nearby is the Plastira Centre Robot Parking - somewhat appropriate for a bunch of developers!

**Comment by papas-source** [comment]
🥇Greek Version
================
Πάντως, tbh, - μια και το είδα ΠΡΩΤΗ φορά το issue στο github - οι περισσότεροι απ' ότι βλέπω λέτε το ΙΤΕ επειδή έχει πρόβλημα parking (!!!) ή είναι δίπλα στη δουλειά σας, στο σπίτι σας, και έχω πάθει ένα σχετικό σοκ!!!

Για όσους αναφέρουν το πάρκινγκ. Θεωρώ ότι μάλλον δεν έχετε κατέβει, γιατί το θέμα του parking είναι αστείο - στο σημείο που είναι το ίδρυμα άντε 50 μέτρα αράουντ, ΠΑΝΤΑ βρίσκεις πάρκινγκ, **ΠΑΝΤΑ ΠΑΝΤΑ ΠΑΝΤΑ**, κυρίως καθημερινές βράδυ που όλοι **ΦΕΥΓΟΥΝ** από το κέντρο. Και στο φινάλε, δίνεις και 3€ σε ένα από τα δύο parking που έχει εκεί δίπλα  αντί να τα δώσεις σε βενζίνες. Άσε που πιστεύω ότι αν πάμε και τους πούμε ότι θέλουμε το parking 10 άτομα, θα μας κάνει ακόμα καλύτερη τιμή.

Όσο για το θέμα του σπιτιού και της δουλειάς μας που "πέφτει" κοντά... Δεν έχω να απαντήσω κατι μια και προφανώς είναι θέμα που από τη μια είμαι σίγουρος ότι δεν είναι σημαντικό για αυτους που το ανέφεραν και από την άλλη, ΟΚ, είναι ΖΗΤΗΜΑ ΠΡΟΣΩΠΙΚΟΥ ΒΟΛΕΜΑΤΟΣ κάτι που σαφώς δεν μπορεί να θεωρηθεί κριτήριο για την επιλογή χώρου. Αλλά αν τολμήσεις ποτέ σασλή να μου ξαναπεις να φύγω από το comfort zone μου, θα σε μπαλοτάρω μα το Θεό.

Αυτό που δεν καταλαβαίνουμε όλοι εμείς (που έχουμε αυτοκίνητα, μηχανάκια κλπ), είναι πόσο δύσκολο είναι για κάποιον πιτσιρικά ή φοιτητή ή εργαζόμενο που τελειώνει εκείνη την ώρα τη δουλειά, να έρθει στο ΙΤΕ. Πέρα από το θέμα του μεταφορικού μέσου, το οποίο σαφώς και είναι το σημαντικότερο, είναι ότι απαιτείται και άλλη μια ώρα πήγαινε - έλα, οπότε κάποιος είναι αναγκασμένος να λείπει 4 - 5 ώρες από το σπίτι του... 

Τώρα... κάποιες πιθανές λύσεις για να έρχεται περισσότερος κόσμος, δεν έχουν εφαρμογή, πχ το να "παίρνουμε" κάποια άτομα με το δικό μας μεταφορικό μέσο. Πρώτον, θα πρέπει αυτός ο κάποιος που δεν έχει αυτοκίνητο να έρθει στο σπίτι μας για να μπορέσουμε να τον πάρουμε - που ξέρετε ότι δεν πρόκειται να γίνει γιατί κανένας δε θέλει να γίνεται "βάρος" σε άλλους (δεν είναι και κολητοί μας, εξάλλου, οπότε πιθανότατα αισθάνονται βάρος) ή να είναι στη διαδρομή μας. Εκτός αυτού, ένας που δε μας ξέρει, δεν πρόκειται να πει "πάρτε με και εμένα σε ένα αυτοκίνητο". Οποιοσδήποτε γνωρίζει βασικές αρχές ψυχολογίας μπορεί να το βεβαιώσει αυτό. Για πολλούς λόγους. Και επίσης δεν θεωρώ σωστό να τους βάζουμε σε αυτή τη διαδικασία...

Όλα αυτά, έχουν αντίκτυπο και στον χρόνο ομιλιών. Όταν ο άλλος έχει χάσει μια ώρα, και θα χρειαστεί και άλλη μισή ώρα να πάει σπίτι του, ας μην απορούμε όταν μας λένε "είμαστε κουρασμένοι, τελειώνετε". Και στο "δέσιμο" της ομάδας. Μετά το ΙΤΕ, είμαστε για σπίτια. Μετά το κέντρο, πίνουμε και καφεδάκι η τρώμε σε ΝΤΕ - ΤΕ.

Επίσης, όλη αυτή η "ψηφοφορία" είναι λίγο αστεία, δεν βλέπω κανένα εκτός από "εμας" να ψηφίζει 


🥇English version
============
I vote for anywhere besides ITE/FORTH

**Comment by kounelios13** [comment]
I vote for 2(Koinonikos xoros)

Ο λόγος είναι ο εξής:

Οι περισσότεροι που ψηφισαν το ΙΤΕ το έκαναν απο οσο βλέπω λόγω του πάρκινκ
Εγω και οι πολλοί φοιτητές δεν χρησιμοποιούμε αυτοκίνητο αλλα λεωφορείο για τις μετακινήσεις μας(shame on us).Ετσι είμαστε αναγκασμένοι όταν τελειώνει το meetup να περιμένουμε και άλλο λεωφορείο ότι ώρα του καπνίσει(Ειναι αρκετά μεγαλο πρόβλημα ιδιαιτερα όταν ο καιρος ειναι κακός).Πολλές φορές έχω χάσει meetups λόγω της απόστασης για το ΙΤΕ(μένω χανιόπορτα που ειναι αρκετα μακρια).Επίσης πιστεύω πως τουλάχιστον οι περισσοτεροι σε περιπτωση που ξαναπάμε στα ιδρύματα καλοκαιρινού θα μπορουν να έρθουν με τα πόδια(δεν λέω για αυτους που μενουν στην κνωσσου,αμμουδαρα αλικαρνασσό και γενικότερα περιοχες μακρια απο το κεντρο) και να γλυτώνουν και βενζινη ;)

**Comment by kounelios13** [comment]
@papas-source  με κάλυψες πιστεύω με το θέμα των φοιτητών :)

**Comment by mvasilak** [comment]
I vote for 2 or anything else near the city center.

**Comment by gsaslis** [comment]
First, thanks for the extra input! it's good to see more ppl voicing their opinions here, so we can finally get some momentum going to change things around... (Remember - you have to be active for this community to work! )

Having said that, let me clarify a couple of things: 

1. The options are 2 - until someone comes up with a third one (btw, thanks @jamiedarcher for offering to help on that front). There are some requirements at the top. We are discussing 1️⃣ vs 2️⃣ here. The discussion is NOT "town centre" vs. "outskirts". Please keep that in mind when commenting below. 

2. As you can see, parking is NOT part of any requirements for the venue (exactly because we never considered or anticipated all our community members have cars). I don't think any comments wrt parking have much to offer to the discussion. Yes, we can consider it in the pros/cons, but first let's find venues that can cater to the requirements. 

3. The problem with 2️⃣ - as you can see above - is how well it covers the requirements. Both projector + internet were a noticeable problem when we were there. If someone has suggestions on how they can be overcome, imho _that_ would be a great contribution to this discussion, so we have two on-par options.

**Comment by DaKnOb** [comment]
Currently OpenCoffee Heraklion is organizing everything at NHMC. In the city center, parking right outside, etc. I've contacted them and received no reply with the previous administration. They have an amphitheater with a capacity of 80, and another room without chairs (many at least) that can fit > 100 people. Also lots of room to bring food, etc.

Maybe we can ask OpenCoffee for contact details if you're interested. It involves manual labor (bring and take out the chairs) and depending on the visitors also find some extra, but we can talk to the OpenCoffee people and ask how they do it..

However, they don't have a speaker / projector installation either in the > 100 people room.

About the Kalokairinou venue, the sound is probably okay thanks to the microphone and speakers they have, but the room can't fit many, and in addition to that we have the issue of the projector and Internet access. About the projector the only solution is to carry one with us, which is difficult, and will have issues in the set up (where it will be, cable length, etc.)

The Internet is a different story. They have an OTE ADSL, which maxes at ~8 Mb/s from what I remember, exactly on the opposite of the speaker stage. As you can image, it's not suitable for anything, since any device amount over 8 will overwhelm it, and also even without anyone else, video calls are not really possible. A few years ago there was CommonsFest, an event that was organized there, and I was in the team responsible to improve the connection. What we did addressed the issue, but it involved lots of cables, hanging from roofs, visits to random people's houses, etc. and is not something that can be done every month.

-- -- -- 
TL;DR
New option: NHMC
About Kalokerinou: Internet can't be fixed; projector issue still remains

**Comment by papas-source** [comment]
Sorry if I am the only one with the question, but... what is this "NHMC"?

**Comment by gsaslis** [comment]
I am reviving this issue. Our current venue (Pancreta Bank) has a serious problem with the internet connection but we have been promised that will be solved in the next few months. 

In addition, we have not seen any significant difference in attendance since moving to a venue in the town center, so I am not sure we have actually helped make our meetups more accessible. In fact, I am concerned we might have done the opposite...  

Should we move back to FORTH? 
Should we explore other options in the town center?
Should we stay out our current venue?

**Comment by sourcegr** [comment]
Hey George, it is always nice to elaborate an look for something better, so yes, reviving this issue is good after all

> we have not seen any significant difference in attendance since

Is this actually based on some measures or is it like the `feels like`? Do we have any real numbers?


>  In fact, I am concerned we might have done the opposite

I don't know what is the basis of this, can you elaborate on why you say that?


> Should we move back to FORTH?

My opinion: no


> Should we explore other options in the town center?

My opinion: we should ALWAYS explore other options


> Should we stay out our current venue?
My opinion: yes - nothing changed for me so see previous comments for the justification

**Comment by kounelios13** [comment]
I agree with the last comment of @sourcegr .I believe the last paragraph is where the drop lies.I think we should give some more time to our current venue

**Comment by DaKnOb** [comment]
I personally believe that the only technical problem with Pancretan is the Internet connection, and it is something that will be fixed, probably before the November Meetup..
For this reason, and given its benefits:

- Location
- Fridge
- Large Room
- Projector, Audio Setup
- No time limitation, from what I know
- Large room outside for networking, unlike FoRTH
- We may not be able to go back if we leave

I propose that we **stay**, but I would also like to see more people being able to book the venue, and not rely on a single person.

**Comment by gsaslis** [comment]
agreed. there could be any number of reasons that would explain a _drop_ in participation. 

but we have had, not one, not two... but 5 (five!) meetups there, so I am not convinced we just happened to have 5 topics (in a row) with lower participation... please remember that topics are voted for by the community itself and so are - by definition - the most popular ones at that point in time.

**Comment by estavrak** [comment]
I'm going to vote for *ITE*.

Apart from the net connection issues at Pancretan, ITE just feels better and I consider it to be closer to the open culture of the community.

I would also like to think that we are not locked in with any venue.

**Comment by sonaht** [comment]
I did not go into the historical messages of the venue issue, but I do remember that the distance (ITE was/is far away) was an issue. We did explore a number of alternatives and we kinda decided to go with the Pancretan Venue, for a good reason, namely the fact that it is just around the corner from the bus terminals etc. So the proximity of the venue was resolved. 

The car parking ability is not the best but with a bit on nonGreek attitude (I wanna park in the front door thingy) there are plenty of spots just around the corner - the old bus terminal. This as an issue is resolved.

Then we realised that internet access is limited due to whatever bank regulation and/or non forward planning at the moment. The problem was brought to the attention of the people in charge and it so seems that an updated internet connection (with a dedicated access - do not quote me on that, Nikos knows a lot better) is getting installed soon. So IMHO this is not an issue anymore, or will be resolved soon, as in should not be an issue.

Apologies for being reactive to the above messages (we all are) the drop or not in attendances is a major issue to discuss and has nothing to do with the venue. We are running for 3 years now... think about this, people had the time to finish Uni, do the army (oops σποντα εδω) and be on their merry way to a brilliant work carrier. Things change, people change, situations change etc etc. 

If you want to discuss attendances lets, but lets also separate issues.

I propose we **STAY**.

**Comment by nickorfas** [comment]
I totally agree with @DaKnOb 

The availability of internet connection for the November's meetup is something that should be confirmed by the bank though.

I propose we **STAY**.

**Comment by sourcegr** [comment]
> but we have had, not one, not two... but 5 (five!) meetups there, so I am not convinced we just happened to have 5 topics (in a row) with lower participation...

Apart from the fact that 3 of them were in summer, have a look at them
* GDPR (acctually, this one was a success)
* IDEs: Super-charging your productivity ()
* Prototyping in VR
* Lightning Talks
* Remote working

Not at all interesting for the majority of the community! Not one - at least in my point of view.

> please remember that topics are voted for by the community itself and so are - by definition - the most popular ones at that point in time.

Yes so? The fact that one topic επικρατεί over oneother, is a metric as to where it stands among those who voted. Not the whole

**Comment by sstauross** [comment]
I vote to stay in PCB if the internet becomes ok as mentioned for big events(over 50-60 people willing to come) due to distance and space for networking after each event. Moreover there are more options after each event to get around with each other. 
I would also propose Forth to stay our venue for smaller events (30-50 people) or hack sessions.

**Comment by svelon** [comment]
I vote for FORTH, only because I feel it is easier to get access to (organizer-wise). I have no problem with staying at PCB though.

**Comment by mzampetakis** [comment]
From my percepctive Pancretan Bank has only one diasadvantage - the insternet connection - which is going to be resolved very soon!

It is at the center of the citym but at the same time there are many parking spots available really close. The place can accomodate many participants where at FORTH some times the room was full (not only at botcoin !). Fridge , luxurious environment, and adequate space for the networking part are also some advantages of the PCB.

I cannot agree the moving from FORTH to PCB decreased the participants as many parameters affect on this and can't be esimated! Also it's good to have a "standard" place for all the meetups , irrelevant to participations, in order to be more familiar to more and more people.

So for me is a STAY!

**Comment by sourcegr** [comment]
Παιδιά, το μεγαλύτερο πλεονέκτημα της PCB είναι ότι μετά το meeting πάμε σε 5 λεπτά και τρώμε όλοι μαζί ή πίνουμε μια μπύρα ή ένα καφέ... Εκεί γίνεται το πραγματικό socializing και δέσιμο της κοινότητας.

**Comment by kapolos** [comment]
Συμφωνώ με τον @estavrak αλλά και με όσους θεωρούν μπελά την επιστροφή από το ITE (επειδή όντως είναι). Tough call.

Είναι δεδομένο ότι είναι αδύνατον να οργανωθεί κάτι ώστε να διευκολύνουμε την επιστροφή; Δεν φαντάζομαι ότι γυρίζουν όλοι στην πόλη με γεμάτα αμάξια.

**Comment by gsaslis** [comment]
I agree with @estavrak not just because it’s a “bank” but also for any corporate-backed venue.

having spoken to plenty of folks in these past years around this topic, I have come to realise there is always an implicit link they make between any event and its venue. (I have been told several times that devstaff was organized by...FORTH itself, just because that was where the meetups were taking place).

We, here, may all indeed know that it’s just a space for the event, but those not so actively involved may not and that’s something I care about. I don’t want our open / volunteer-driven community to be “coloured” as a corporate initiative.

And this is not just a matter of fairness or principle. There are several implications. For example:
* a competitor’s employees would be less likely to attend,
* other sponsors might be discouraged from supporting,
* people who don’t like the specific corporate / business may decide against participating (ok, this could apply to any space but is much less likely to apply to public spaces).

Don’t get me wrong... PCB offers an excellent venue (-internet) and has also made my life easier, but I’ve come to understand there's more to organizing events than just that

**Comment by gsaslis** [comment]
(Closed by misclick...doh...)

In any case, we _do_ need another venue for the time being (in November’s meetup we’ll have a remote speaker), but I would very much like to see the discussion continue and am happy to go with the majority vote on this

**Comment by sourcegr** [comment]
Well, I don't want to be harsh, but these are all non issues, rants, speculations and contadictive thoughts.

You assume that people will think that way. And even if we - kind of - make these people happy (according to their view on happiness) we could at the same time make some others unhappy (according to reality) eg: they dont have the means to drive to ITE.

- Not having a car, is reality
- Not being able to spend 1 hour driving/on the roads, is a reality.
- Thinking that holding a meeting at a bank provided venue is a bad think, is a matter of perspective.
- Thinking that the bank organizes the event - although it is clearly stated that it is not - is a matter of (wrong) perspective.

We can debate all day for matters of perspective, since everybody has one. I'd prefer we stick to the facts.


Now, because I need to pinpoint your contradictory thoughts...

> I have been told several times that devstaff was organized by...FORTH itself, just because that was where the meetups were taking place

Right! But I bet you were never told that BCB organizes the events now that we moved...
So, in reallity, not doing our meetings to ITE is good for our community. It makes it CLEAR that DEVSTAFF is an entity and DEVSTAFF organizes it.

Think about Papaki who is our great sponsor (and thank god for that!). Would it be strange if someone thought that papaki organises all these? Not at all. I for myself thought about it when I first heard about the meetings.
Think about the office12 where we mostly host our hack sessions.  Would it be strange if someone thought that office12 organises all these?

> a competitor’s employees would be less likely to attend

If this applies to PCB, it would also apply to the competitors of papaki and office12. So, you are being picky here just to make a point.  I tell you that an employee of the X company (that is a competitor to one of our sponsors), would also be less likely to attend -according to this logic

> other sponsors might be discouraged from supporting

You either accept sponsors (PCB is one of them) or ditch them completely.

**Comment by gsaslis** [comment]
> You assume that people will think that way.

I am not. I guess it wasn't clear (even though I did start with "having spoken to plenty of folks"...) that I was actually passing on feedback I _have_, in fact, received from attendees. I take time to speak to people in our events (including - in particular - people I have never met before & also newcomers to events) and make an active effort to collect feedback from them and have been doing so for the past 3+ years, so, no, in this case, there is no assumption involved. 

If someone else wants to claim they have done the same and that they can speak for more than themselves, I am happy to compare the feedback we have each individually collected. 

> we could at the same time make some others unhappy

I never suggested anything else than a public vote that everyone registered on meetup.com will have a right to participate. So, "we" would in fact not be making anyone anything. This is a choice everyone would be making for themselves. 

> Right! But I bet you were never told that BCB organizes the events now that we moved...
So, in reallity, not doing our meetings to ITE is good for the brand of our community. It makes it CLEAR that DEVSTAFF is an entity and DEVSTAFF organizes it.

1) ☝️ argument based on speculation ("I bet you were never told") 
2) ☝️contradiction (we will either consider the 5 meetups at PCB as a sufficient sample to draw some conclusion from, or we will not. a few comments up, it was suggested that we shouldn't, because it's not representative).

On sponsors, I think your point on all vs. nothing has some validity, and it has come up way back, when discussing the matter of sponsors altogether (for some background: for the first few meetups we actively rejected sponsorships - and for more detailed history, please read https://github.com/devstaff-crete/DevStaff-Heraklion/pull/192, https://github.com/devstaff-crete/DevStaff-Heraklion/issues/4 and [DevStaff-Heraklion/Sponsors.md](https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/Sponsors.md) ). 

> If this applies to PCB, it would also apply to the competitors of papaki and office12. So, you are being picky here just to make a point. I tell you that an employee of the X company (that is a competitor to one of our sponsors), would also be less likely to attend - according to this logic

Indeed, you are right, this was one of the concerns back then and I was also one of the people supporting it. 

However, what I also tried to explain above (unsuccessfully, it seems) is that the place/venue is not the same as sponsorship for pizzas.

**Comment by kounelios13** [comment]
@kapolos Όσον αφορά το attendance drop.Ισως απλά τα θέματα των τελευταίων meetups είχαν λιγότερο ενδιαφέρoν σε σχέση με αυτά που γινόταν στο ΙΤΕ.Νομίζω οτι θα πρέπει να γίνουν μερικά ακομα στην παγκρήτια για να μπορεί να υπωθει με σιγουρια ότι φταίει το venue για το drop

**Comment by sourcegr** [comment]
@kapolos ήξερα ότι θα ακουστώ έτσι, για αυτό προετοίμασα το έδαφος 😆 

Για το  attendance... Το εξαντλήσαμε στα προηγούμενα σχόλια νομίζω, οπότε ας μην τα ξαναλέω.
Για το brand... εξαρτάται, αν θες να νομίζουν όλοι ότι είσαι σπιναπ του forth, ναι είναι καλό. Αν θες όμως να είσαι το δικό σου brand, όχι δεν είναι.

@gsaslis εξήγησα γιατί θεωρώ ότι είναι άκυρα τα points σου. Απο κει και πέρα, είναι λογικό ο καθένας να βλέπει διαφορετικά τα πράγματα. Αν και δεν έχει νόημα να το εξηγήσουμε περαιτέρω, να ξεκαθαρίσω ότι το 

> You assume that people will think that way

Αναφερόταν στα bullet points σου, που ξακάθαρα γράφεις "would be", "might be", "may decide" τα οποία **είναι** assumptions/speculations. Κανένας δεν έχει αμφισβητήσει το ενδιαφέρον σου για το devstuff - και αν κάποιος το κάνει πρώτα εμένα θα βρει απέναντί του.

Επίσης για το (2☝️) Eγώ δεν είπα πουθενά ότι πιστεύω πως χρειάζεται χρόνος το venue για να δούμε αν αυτό φταίει. Δεν πιστεύω ότι φταίει αυτό και εξήγησα επακριβώς τι νομίζω ότι φταίει. Συνεπώς μην το παρουσιάζεις σαν δικό μου contradiction.

Για το (1 ☝️ ) Ειλικρινά θα βοηθούσε να μας πεις αν κάποιος σου είπε ότι νόμιζε πως το meeting το οργάνωσε η τράπεζα. Χωρίς να εξετάζουμε αν είναι valid σαν σκέψη - που όλοι ξέρουμε ότι δεν είναι. Έχει συμβεί κάτι τέτοιο;

Και αν ναι, γιατί πρέπει να αποδεικνύουμε ότι δεν είμαστε ελέφαντες;

**Comment by kapolos** [comment]
@kounelios13 Προσωπικά συμφωνώ απόλυτα με αυτό που λες. Πράγματι, ενώ μπορεί να μην είμαστε σε θέση να ξέρουμε ποιοι είναι ακριβώς οι λόγοι και την βαρύτητα τους, είναι όντως τόσο απλό όπως το περιγράφεις. Ν meetups αργότερα στον ίδιο χώρο, αν το attendance συνεχίζει να υπολείπεται είναι προφανές πείραμα το να γίνουν μερικά στο ΙΤΕ και να συγκρίνουμε. Αν δεν υπάρχει διαφορά, αλλού βρίσκονται οι λόγοι αλλά τουλάχιστον θα ξέρουμε σίγουρα ότι δεν φταίει ο χώρος.

@sourcegr Θεωρώ ότι το group επιτυγχάνει τον ρόλο του καλύτερα όσο μεγαλώνει το attendance, μιας και έτσι υπάρχει μεγαλύτερη διάχυση πληροφορίας και εντελώς απλά, γινόμαστε χρήσιμοι σε πιο πολλούς. Υπό αυτή την οπτική, **αν** το branding του Forth κάνει brush off στο DevStaff και αυτό φέρνει περισσότερο κόσμο, νομίζω ότι συμφωνούμε όλοι ότι είναι θεμιτό (χωρίς με αυτό τον τρόπο να υπονοώ ότι τα προβλήματα της απόστασης δεν είναι σημαντικά και δεν χρειάζεται να το δούμε σοβαρά).

**Comment by nickorfas** [comment]
It is confirmed by the PCB that we can have an upgraded internet connection available up until our November's meetup (8/11/18).

It was said that - at this point - internet access will be given to those that actually need it (I suppose, for example, to those that will use their laptops in order to make Skype calls or whatever else required in order to make participating or speaking remotely possible). 

I did not clarify if later on they plan to give internet access to everyone that participates in the meetups, but I believe that we can ask them in the future if we think it is a critical issue.

**Comment by sourcegr** [comment]
Καλημέρα. 

Το Η2Β ακούγεται καλή ιδέα αρχικά.

Αλλά... κατάλαβα καλά; Χωράνε μέχρι 50 άτομα; Γιατί αν ναι, πολύ φοβάμαι ότι είναι πολύ μικρής χωρητικότητας. 50 άτομα  μαζεύουμε ακόμα και αν κάνουμε meetup για την επίπτωση που έχουν τα semicolons στην JS στον αμφιβληστροειδή της γάτας του developer...

Κατά τα άλλα, το parking εκεί, είναι ΠΡΑΓΜΑΤΙ πρόβλημα...

**Comment by mpof7** [comment]
Hi everyone,

I don't usually participate in such "debates - discussions" but I thought "Why not?". What a vivid conversation you have here! Okay, my vote goes to H2B HUB because it comprises something new (and thus neutral) and also it's in the town center. Having said that, I should mention that in my opinion the "attendance drop" issue is not only related to the venue and I think that the organizers firstly and the community as whole secondly, should take into consideration the following aspects - operations:

- Subject choice (voting is not the only efficient method, variety of subjects???)
- Reckoning and mapping the audience (get to know your audience and especially the "newcomers")
- Delivery method(s) (merely through static presentations and discussions only, more interesting methods available to assist)
- Social networking (as organizers try to help people to commute; again especially the newcomers, maybe through some "tricks" or initiatives regarding the "free hour" and apart from pizzas and drinks)
- Take into consideration to reduce the time from 3 to 2 hours OR incorporate some brakes (not everyone have the same energy levels as well as the ability to continuously concentrate on different subjects)

P.S. Try to make the DevStaff Community meetups less typical, boring and strict, instead make them more interesting, engaging, useful and funny to the members.

Thanks

**Comment by zakkak** [comment]
Just an update regarding the FORTH room. It has been renovated (and it looks pretty nice, but that's not that important) and is now equipped with a nice projector with embedded speakers (and finally you can use HDMI to connect to it).

**Comment by sonaht** [comment]
Guys shall we close this down... it is fairly covered and no point having still open.

**Comment by sonaht** [comment]
closing this as very old, see title
